src/main.c

```diff
@@ -390,7 +390,9 @@ int main(void) {
 	int fd = sceIoOpen("ux0:email/message/mail.db", SCE_O_RDONLY, 0);
 	if (fd < 0) {
 		printf("It seems that the mail database does not exist (open ux0:email/message/mail.db err=0x%x)\n", fd);
-		printf("You should open the Email app from LiveArea and then close it without adding an account.\n");
+		printf("You should open the Email app from LiveArea and then close it without adding an account.\n", fd);
+		printf("If you still see this error message, be sure that you've enabled the "Unsafe Homebrew" option inside molecularShell.\n", fd);		
```

Comment:
Seems like a syntax error here. Please make sure the code compiles before submitting a PR.